common/common-ui/src/main/res/layout/dialog_text_alert.xml

```diff
@@ -52,50 +54,28 @@
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toBottomOf="@+id/textAlertDialogTitle"
         app:textType="secondary"
-        app:typography="body1"/>
-
-    <com.duckduckgo.common.ui.view.button.DaxButtonPrimary
-        android:id="@+id/textAlertDialogPositiveButton"
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:layout_marginTop="@dimen/keyline_4"
-        android:text="Positive"
-        app:layout_constraintBottom_toBottomOf="parent"
-        app:layout_constraintEnd_toEndOf="parent"
-        app:layout_constraintTop_toBottomOf="@+id/textAlertDialogMessage"/>
+        app:typography="body1" />
 
-    <com.duckduckgo.common.ui.view.button.DaxButtonDestructive
-        android:id="@+id/textAlertDialogPositiveDestructiveButton"
-        android:layout_width="wrap_content"
+    <CheckBox
+        android:id="@+id/textAlertDialogCheckBox"
+        android:layout_width="0dp"
```

Comment:
_Question_: why `0dp` instead `wrap_content` if we are not setting the end constraint?

Reply:
No reason, will revert because that makes sense.